src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,77 @@
+package wolox.training.models;
+
+import com.sun.istack.NotNull;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+@Table(name = "users")
+@Entity
+public class User {
+
+    public User() {
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private long id;
+    @NotNull
+    @Column(nullable = false)
+    private String username;
+    @NotNull
+    @Column(nullable = false)
+    private String name;
+    @NotNull
+    @Column(nullable = false)
+    private LocalDate birthdate;
+    @ManyToMany
+    private List<Book> books = Collections.emptyList();
```

Comment:
When adding a many to many relation we need the other part of the relation. Now we should add the field into the book model.

  @ManyToMany (mappedBy = "books")
  private List <User> user;
Remember to add its respective get as you did in this collection.

Additional a good practice is add {CascadeType.MERGE, CascadeType.REFRESH}), although it is not mandatory for this card